build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
Also to be clear, 3 Core_Roots are uploaded - 1 for Win, 1 for Linux, 1 for OSX. All Linux jobs would be crossgening the same Core_Root, and all OSX jobs would be crossgening the same Core_Root.